### PR TITLE
Fix focussing to internal link after copy

### LINKED ISSUE #34281: [Bug]: copying internal links does not work (error: "cannot copy, please copy the link manually")
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

[Nextcloud 25] Copying internal links in Nextcloud Files does not work with Chrome or Safari on Mac OS. I tried it with a few different files (.ods, .md). If you click on the copy button, nothing happens. If you hover with your cursor over it, it will show "Cannot copy, please copy the link manually".

Screenshot-2022-09-27-at-10 34 16

### Steps to reproduce

1. Click on the "Internal link" icon in the sidebar of a file you want to share.
2. The link is not copied, if you hover of it with your cursor it shows "Cannot copy, please copy the link manually"

### Expected behavior

The internal link will be copied and can be pasted.

### Installation method

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database engine version

_No response_

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

_No response_

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
cloud.nc.com / not sure
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

Using Google Chrome 105.0.5195.125 (Official Build) (arm64) on macOS Monterey 12.6 or Safari Version 16.0 (17614.1.25.9.10, 17614).

Console output:

SharingEntryInternal.vue:94 TypeError: Cannot read properties of undefined (reading '$el')
    at SharingEntryInternal.vue:88:1
    at u (runtime.js:63:40)
    at Generator._invoke (runtime.js:294:22)
    at Generator.next (runtime.js:119:21)
    at q (SharingEntrySimple.vue?cb12:20:33)
    at s (SharingEntryInternal.vue:84:1)
(anonymous) @ SharingEntryInternal.vue:94
u @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
q @ SharingEntrySimple.vue?cb12:20
s @ SharingEntryInternal.vue:84
Promise.then (async)
q @ SharingEntrySimple.vue?cb12:20
s @ SharingEntryInternal.vue:84
(anonymous) @ SharingEntryInternal.vue:84
(anonymous) @ SharingEntryInternal.vue:84
copyLink @ SharingEntryInternal.vue:84
click @ SharingEntryInternal.vue?vue&type=template&id=b5df6444&scoped=true&:1
click @ NcActions.js:2
Ct @ vue.runtime.esm.js:2987
n @ vue.runtime.esm.js:1785
Pa.r._wrapper @ vue.runtime.esm.js:7428

### PR DESCRIPTION
**Pull request by PVince81**
Since a recent change to NcActions it's not possible to add a "ref" attribute on a NcAction* component.

As a workaround, a ref was added on the NcActions parent component instead.

Fixes https://github.com/nextcloud/server/issues/34281

### CODE REVIEW

File: apps/files_sharing/src/components/SharingEntryInternal.vue [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+			class="sharing-entry__internal"
 			:title="t('files_sharing', 'Internal link')"
 			:subtitle="internalLinkSubtitle">
 			<template #avatar>
 				<div class="avatar-external icon-external-white" />
 			</template>
 
-			<NcActionLink ref="copyButton"
+			<NcActionLink
 				:href="internalLink"
```
Comment (nickvergessen) [inline apps/files_sharing/src/components/SharingEntryInternal.vue:13]:
```suggestion
			<NcActionLink :href="internalLink"
```

**Comment by skjnldsv** [comment]
> Since a recent change to NcActions it's not possible to add a "ref" attribute on a NcAction* component.

That's a regression. This is one of the basic feature of Vue. It means we are doing something wrong :confused:

**Review by skjnldsv — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
> > Since a recent change to NcActions it's not possible to add a "ref" attribute on a NcAction* component.
> 
> That's a regression. This is one of the basic feature of Vue. It means we are doing something wrong 😕

It's a regression of https://github.com/nextcloud/nextcloud-vue/pull/2911. I created a ticket with the explanation in https://github.com/nextcloud/nextcloud-vue/issues/3298.

**Review by raimund-schluessler — APPROVED** [review]
Code looks good, I didn't test it, though.

**Comment by raimund-schluessler** [comment]
I created https://github.com/nextcloud/nextcloud-vue/pull/3299 which helps to address this problem.

However, even with a fix of nextcloud/vue, server would still need adjustments, since we use a render function now, which means the `ref` `copyButton` would not be available in the context of the `SharingEntryInternal` component anymore, but only in the context of the `NcActions` component.

**Comment by blizzz** [comment]
master is 26, please trigger backport if necessary

**Review by Pytal — APPROVED** [review]

**Pull request merged**